cornac/models/upcf/requirements.txt

```diff
@@ -0,0 +1 @@
+similaripy
```

Comment:
I'm not sure if this library is well-maintained and will be backward compatible for future versions. Maybe we can pin a specific working version?

Reply:
Updated.